docs/01-start-here/08-incidents.md

```diff
@@ -0,0 +1,150 @@
+
+Check the number of EC2 instances for the affected apps, and what state they are in. 
+You can see this from the [CloudWatch overview board](https://eu-west-1.console.aws.amazon.com/cloudwatch/home?region=eu-west-1#dashboards:name=xOverview) which has a chart for number of healthy/unhealthy 
+instances over time. If there are no healthy instances the app may be failing to start.
+
+## Judge whether to scale up, roll back
+
+### Scaling Up
+
+In nearly all cases, scaling up an app is a good interim solution to problems in
+general. There are only two cases where you may not want to scale up: when apps
+are failing to even start (new ec2 instances never become healthy), or when an upstream
+or downstream dependency is the cause of your problems (in these cases scaling up
+may cause that dependency to come under even more load).
+
+If you are seeing an increase in REAL traffic (i.e. not just due to Fastly making 
+more requests because your app is not returning 200s), you should scale up.
+
+If instances are maxing their CPU (as seen in the xApp CloudWatch dashboards) you 
+should scale up.
+
+If you are seeing high latency, no application errors in particular, and new instances 
+are coming up healthy, you should scale up.
+
+### Rolling back
+
+If you are seeing application errors in Kibana that point to a software bug, you 
+should roll back by using Riff-Raff to deploy a previous build.
+
+## Pinpoint an exact cause
+
+### For traffic related issues
+
+If you are experiencing high user traffic in general, this may be due to an increased
+number of cache misses on fastly. You can see the cache hit rate on the [Fastly dashboard](https://manage.fastly.com/)
+
+You might be experiening a single user flooding the app with thousands of requests. 
+You can drill into the source-api chart on the [Kibana Overview Board](https://logs.gutools.co.uk/app/kibana#/dashboard/00349ef0-06a1-11e8-a56d-a31118fab969?_g=(refreshInterval%3A(display%3AOff%2Cpause%3A!f%2Cvalue%3A0)%2Ctime%3A(from%3Anow-15m%2Cmode%3Aquick%2Cto%3Anow)))
+to see if this is the case.
+
+### For High Latency without high traffic
+
+If your apps are mysteriously slowing down, this could be due to an upstream/downstream
+dependency, such as CAPI, becomming slow. You can see the CAPI team dashboard [here](http://status.capi.gutools.co.uk/).
+Our own Kibana dashboards _also_ show CAPI latency, but it can be misleading as it 
+incorporates our own latency into the measure, in other words if we are slow talking to 
+CAPI, then it will be reported as 'CAPI being slow' even though the latency is within our own 
+app.
+
+### Analysing Instances Directly
+
+If you suspect that there is a real issue with an app you can connect directly to the instance
+using to try and collect more runtime information. 
+
+You should start by picking an instance and detaching it from the load balancer so that it doesn't
+get killed or interfered with, and then connect to via ssm. 
```

Comment:
Maybe it's worth running top/lsof before detaching from the ELB - to see what the app looks like under load

Reply:
good point